34925 Help windows on pop up boxes cannot be closed

tyndmar opened the pull request:
sys/ref/oldhlpd.w - the window version of help, caused original problem
sys/ref/hlpd.w - a dialog version of help, with programmer functions the start other dialogs removed
util/wPgmrToolbox.w - dialog that replaces the extra functions formerly in help (F3) screen - accessible from main menu by pressing Alt-Ctrl-P
system/mainmenu2.w - added trigger to launce pgmrtoolbox
system/PgmMstrSecur.w - revised to account for additional functions in pgmrtoolbox.

Commit:
34925 Help windows on pop up boxes cannot be closed
sys/ref/oldhlpd.w - the window version of help, caused original problem
sys/ref/hlpd.w - a dialog version of help, with programmer functions the start other dialogs removed
util/wPgmrToolbox.w - dialog that replaces the extra functions formerly in help (F3) screen - accessible from main menu by pressing Alt-Ctrl-P
system/mainmenu2.w - added trigger to launce pgmrtoolbox
system/PgmMstrSecur.w - revised to account for additional functions in pgmrtoolbox.

brad-advantzware reviewed (changes requested):
PggMstrSecur.p - wipes out changes - need to correct

Commit:
Merge branch 'develop' into feature/34925_Help_windows_on_pop_up_boxes_cannot_be_closed

Commit:
34925 Help windows on pop up boxes cannot be closed
datadigger.ini - admin only, automatic changes
system/PgmMstrSecur.p - readded Sewa's change that was inadvert. left out, alphabetized lists for easier future access
util/wPgmrToolbox.w - fixed test scenario for db monitoring (l. 354,380), improved security checking (l. 407-439)

brad-advantzware reviewed (changes requested):
oldhlpld.w still in pull request.
Why is datadigger.ini tracked?
PgmMstrSecur seems that there are a ton of changes but trusting they are consistent with what we reviewed.  Typo in asilogin.w  (system/mainment2.w) - why is this needed anyway?

Commit:
34925 Help windows on pop up boxes cannot be closed

brad-advantzware reviewed (approved):
Approved but do not merge in DataDigger.ini files.  sHould not be included in this

Commit:
Merge branch 'develop' into feature/34925_Help_windows_on_pop_up_boxes_cannot_be_closed
